.gitignore

```diff
@@ -63,7 +63,6 @@ $RECYCLE.BIN/
 # Icon must end with two \r
```

Comment:
as it says here it must end with the two empty lines, do not do random changes like this please